quay-enterprise/prometheus.md

```diff
@@ -0,0 +1,19 @@
+# Prometheus metrics under Quay Enterprise
+
+Quay Enterprise exports a [Prometheus](https://prometheus.io/)-compatible endpoint on each instance to allow for easy monitoring and alerting.
+
+## Exposing the prometheus endpoint
+
+The Prometheus-compatible endpoint on the Quay Enterprise instance can be found at port `9092`. Simply add `-p 9092:9092` to the `docker run` command (or expose the port via the `Pod` configuration in Kubernetes).
```

Comment:
We're using 9092 for the time being; we might change it in the future, but the docs should reflect the current port (for now)